src/org/spdx/rdfparser/license/SimpleLicensingInfo.java

```diff
@@ -78,6 +78,17 @@ public void getPropertiesFromModel() throws InvalidSPDXAnalysisException {
 				this.seeAlso[startExtraIndex + i] = moreSeeAlso[i];
 			}
 		}
+		// SourceUrlDetails/seeAlsoDetails
+		this.seeAlsoDetails = this.findMultiplePropertyValues(SpdxRdfConstants.RDFS_NAMESPACE, SpdxRdfConstants.RDFS_PROP_SEE_ALSO_DETAILS);
```

Comment:
Should `SpdxRdfConstants.RDFS_PROP_SEE_ALSO_DETAILS` be change to `SpdxRdfConstants.LICENSEXML_ELEMENT_CROSS_REF`?

Reply:
I have corrected this